addons/website_sale/static/src/js/notification/cart_notification/cart_notification.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import { Component } from "@odoo/owl";
+import { formatCurrency } from "@web/core/currency";
+
+export class CartNotification extends Component {
+    static template = "website_sale.cartNotification";
+    static props = {
+        message: [String, { toString: Function }],
+        lines: {
+            type: Array,
+            element: {
+                type: Object,
+                shape: {
+                    id: Number,
+                    image_url: String,
+                    quantity: Number,
+                    name: String,
+                    description: { type: String, optional: true },
+                    line_price_total: Number,
+                },
+            },
+        },
+        currency_id: Number,
+        className: String,
+        close: Function,
+    }
+
+    /**
+     * Get the top position (in px) of the notification based on the navbar height.
+     *
+     * This prevents the notification from being shown in front of the navbar.
+     */
+    get positionOffset() {
+        return (document.querySelector('header.o_top_fixed_element')?.offsetHeight || 0) + 'px';
+    }
+
+    getProductSummary(line) {
```

Comment:
This one has no docstring either (will you remove it as well? :smile:)

Reply:
Well I had to try :sweat_smile: